src/styles/app.css

```diff
@@ -1,79 +1,392 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;900&family=Roboto+Slab:wght@300;900&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 body {
-  font-family: "Segoe UI", sans-serif;
-  line-height: 1.4;
   color: #000;
   background: #fff;
-  height: 100vh;
-  font-weight: 400;
+  height: 90vh;
+  font-family: 'Roboto Slab', serif;
   list-style-type: none;
+  text-decoration: none;
 }
 
-.container {
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  align-items: center;
+a {
+  text-decoration: none;
 }
 
 .container-nav {
   display: flex;
-  justify-content: left;
+  justify-content: space-between;
+  margin: 1.813rem 6.25rem 1.813rem 6.25rem;
+  background-color: #fff;
+}
+
+.panel-bg {
+  display: flex;
   flex-direction: row;
-  padding: 2rem;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 2.938rem;
+  background-color: #fff;
+}
+
+.Bookstore-CMS {
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
```

Comment:
* Kindly adjust your styling to suit the Styling provided below in the highlighted screenshot and should be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)
## YOURS 
![image](https://github.com/danielamoreno699/bookstore/assets/52497246/8958d0f8-a626-4bcd-9f24-54c97feaea56)
## DESIGN 
![image](https://github.com/danielamoreno699/bookstore/assets/52497246/793b3f2b-1ee1-40b8-86c4-d86147b47814)